e2e/cypress.json

```diff
@@ -1,8 +1,8 @@
 {
     "baseUrl": "http://localhost:8065",
     "viewportWidth": 1300,
-    "defaultCommandTimeout": 20000,
-    "taskTimeout": 20000,
+    "defaultCommandTimeout": 60000,
+    "taskTimeout": 60000,
```

Comment:
I'm good here as temporary fix only but we need to further investigate whether the issue is application's performance degradation (in terms of first loading and needs to address by Dev) or is related to Cypress framework itself (SDETs to fix).

Reply:
I had checked that the slowness is in the app. For example the posts does not loads within 20 seconds on center channel and it times out and fails the test